SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -322,7 +242,7 @@ bool FLocalDeploymentManager::TryStopLocalDeployment()
 	bool bSuccess = false;
 
 	TSharedPtr<FJsonObject> SpotJsonResult;
-	bool bPasingSuccess = ParseJson(SpotDeleteResult, SpotJsonResult);
+	bool bPasingSuccess = FSpatialGDKServicesModule::ParseJson(SpotDeleteResult, SpotJsonResult);
```

Comment:
The `bPasingSuccess` typo has reemerged 💀

Reply:
:o how strange. I based the branch on the one which was merged with the RC. Hope nothing else has reemerged

Reply:
Down below the `bPasingSuccess` is actually correct - might be just me fixing only one of them

Reply:
I've actually fixed this in a different branch so let's worry about it there :D